.travis.yml

```diff
@@ -2,16 +2,11 @@
 language: generic
 env:
     matrix:
-        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev
-          testflags="--test-type integ --pytest-args='-x'"
         - DOCKER_IMAGE=nmstate/centos8-nmstate-dev
           testflags="--test-type integ --pytest-args='-x'"
-        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev
+        - DOCKER_IMAGE=nmstate/centos8-nmstate-dev
           testflags="--test-type integ --pytest-args='-x'
               --copr networkmanager/NetworkManager-master"
-        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev
```

Comment:
Why have you taken out NM 1.20 tests? 8.0 does not yet include this version, or did we embedded it in the image? (If we did, please mention this in the commit).

Reply:
It is already in the image, updated the commit message.